ffmpeg/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+link="https://ffmpeg.org/releases/ffmpeg-snapshot.tar.bz2"
+
+echo "Downloading FFMPEG source code ..."
+wget -q "$link"
+echo "Unzipping source code ..."
+tar -xf ffmpeg-snapshot.tar.bz2
+cd ffmpeg/ || exit 1
+
+echo -n "Patching ..."
+./configure --extra-cflags="-fPIC" --extra-ldflags="-static-pie"  --disable-shared --disable-opencl
+
+echo -n "Compiling ..."
+make -j
```

Comment:
```suggestion
make -j "$(nproc)"
```
When using `-j` without arguments, it creates a lot of jobs and the system becomes unresponsive. Use `make -j $(nproc)` to have the maximum number of jobs equal to the number of CPUs.